packages/flutter/lib/src/rendering/sliver_padding.dart

```diff
@@ -92,38 +92,34 @@ class RenderSliverPadding extends RenderSliver with RenderObjectWithChildMixin<R
     return null;
   }
 
-  /// The padding in the cross-axis direction on the side to the right when
-  /// facing away from the zero scroll offset in the scroll axis direction. (In
-  /// other words, for a vertical downwards-growing list, the padding on the
+  /// The padding in the cross-axis direction on the side closest to the origin.
+  /// (In other words, for a vertical downwards-growing list, the padding on the
   /// left.)
   ///
   /// Only valid after layout has started, since before layout the render object
   /// doesn't know what direction it will be laid out in.
-  double get startPadding {
+  double get leadingPadding {
```

Comment:
we normally refer to the "leading edge" as meaning the "before" side using BASE terminology, and this edge as the "start" edge. I wouldn't mind replacing "before" with "leading" but replacing "start" with "leading" seems confusing.